frontend/components/YourWalletPage/WithdrawFunds/index.ts

```diff
@@ -0,0 +1 @@
+export { WithdrawFunds } from './WithdrawFunds';
```

Comment:
I thought we agreed not to have index.ts files (tho as I remember somewhere we still do it), curious why you decided to use it here?